API.md

```diff
@@ -1156,22 +1156,26 @@ for more documentation.
         automobile and will use highways, `'walking'`, which avoids
         streets without sidewalks, and `'cycling'`, which prefers streets
         with bicycle lanes and lower speed limits for transportation via
-        bicycle. (optional, default `driving`)
+        bicycle. The `'mapbox/driving-traffic'` profile is not supported. (optional, default `driving`)
```

Comment:
This should just say `driving-traffic` to be consistent with how the other profiles are specified.

The HTTP API uses the full `mapbox/driving-traffic` or `mapbox/driving` as the profile name, but it looks like the SDK automatically pre-pends `mapbox/`, so we probably shouldn't mention it.